package.json

```diff
@@ -29,18 +29,19 @@
     "prettier": "prettier '**/*.+(js|json|ts|md|yml|yaml)'",
     "format": "npm run prettier --write",
     "check-format": "npm run prettier --list-different",
-    "lint": "eslint --ignore-path .gitignore .",
+    "lint": "eslint .",
     "validate": "npm run lint && npm run check-types && npm run check-format",
-    "prepare": "husky install",
+    "prepare": "test -d node_modules/husky && husky install || echo \"husky is not installed\"",
```

Comment:
Needed because git hooks are run even when dev dependencies aren't there (in CI). Ref: [issue](https://github.com/typicode/husky/issues/914)